content/docs/user-guide/components/reference/paintbrush/paintbrush.md

```diff
@@ -167,3 +167,18 @@ The **Smoothing Radius** is separate from the Paint Brush **Size**. The Paint Br
 | `ed_paintBrushHardnessPercentAdjustAmount` | Controls how much the `{` and `}` hotkeys adjust the Paint Brush Hardness in percent (0 - 100). |
 | `ed_paintBrushManipulatorInnerColor` | The color of the inner circle of the Paint Brush, which is used to show where the hardness falloff begins. |
 | `ed_paintBrushManipulatorOuterColor` | The color of the outer circle of the Paint Brush, which shows the overall Paint Brush size and where the hardness falloff ends. |
+
+## PaintBrushSessionBus
+
+The PaintBrush provides an EBus that exposes high-level painting APIs for any component that supports runtime painting.
+
+| Request Name | Description | Parameter | Return | Scriptable |
+|-|-|-|-|-|
+| `StartPaintSession` | Starts a runtime paint session and creates the temporary data buffers needed for modification. | PaintableEntityId: The entity that contains a paintable component.  | `Uuid`: The new paint session ID. | Yes |
+| `EndPaintSession` | End the runtime paint session for the given ID and cleans up the temporary data buffers. | sessionId: The paint session ID to end. | None | Yes |
+| `BeginBrushStroke` | Start a brush stroke for a paint session with the given brush settings for color/intensity/opacity. | sessionId: The paint session ID, brushSettings: The paint brush settings for this brush stroke. | None  | Yes |
+| `EndBrushStroke` | End a brush stroke for a paint session. | sessionId: The paint session ID. | None  | Yes |
```

Comment:
Nit: For consistency, you can choose to add an -s to the end of verb or remove it for each description. There's currently no style rules on which to use, but you can reason for either of the two options: 
- Add -s to a verb with a third-person singular subject. The subject in these cases are each request name.
- Don't add -s to a verb with second-person subject. The subject is 'you' the user. 
([Source](https://www.britannica.com/dictionary/eb/qa/Inflections-When-to-Add-S-to-Regular-Verbs))

I went with option 1 to demonstrate. There are other instances in other pages, though again it's nit, and I'll leave it to your discretion to adjust. 
```suggestion
| `EndPaintSession` | Ends the runtime paint session for the given ID and cleans up the temporary data buffers. | sessionId: The paint session ID to end. | None | Yes |
| `BeginBrushStroke` | Starts a brush stroke for a paint session with the given brush settings for color/intensity/opacity. | sessionId: The paint session ID, brushSettings: The paint brush settings for this brush stroke. | None  | Yes |
| `EndBrushStroke` | Ends a brush stroke for a paint session. | sessionId: The paint session ID. | None  | Yes |
```